test/e2e/framework/util.go

```diff
@@ -605,21 +605,19 @@ func AssertCleanup(ns string, selectors ...string) {
 
 // LookForStringInPodExec looks for the given string in the output of a command
 // executed in a specific pod container.
-// TODO(alejandrox1): move to pod/ subpkg once kubectl methods are refactored.
 func LookForStringInPodExec(ns, podName string, command []string, expectedString string, timeout time.Duration) (result string, err error) {
-	return LookForString(expectedString, timeout, func() string {
+	return lookForString(expectedString, timeout, func() string {
 		// use the first container
 		args := []string{"exec", podName, fmt.Sprintf("--namespace=%v", ns), "--"}
 		args = append(args, command...)
 		return RunKubectlOrDie(ns, args...)
 	})
 }
 
-// LookForString looks for the given string in the output of fn, repeatedly calling fn until
+// lookForString looks for the given string in the output of fn, repeatedly calling fn until
 // the timeout is reached or the string is found. Returns last log and possibly
 // error if the string was not found.
-// TODO(alejandrox1): move to pod/ subpkg once kubectl methods are refactored.
```

Comment:
i think the todo is expired.

Reply:
> maybe move to https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework/pod is a better solution.

I feel the above TODO message `move to pod/ subpkg` means https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework/pod
So I am fine to keep this TODO.

BTW it is nice to make this function to a private one.

Reply:
thanks.
how about the progess `kubectl methods are refactored`?

Reply:
ha :sweat_smile: yeah, by pod subpkg i meant e2epod (/test/e2e/framework/pod).
The rationale for this TODOs is that the `lookForString` (+1 for making it private) function is used by `LookForStringInExecPod`.
The latter is used in tests along with other functions from e2ekubectl (/test/e2e/framework/kubectl).
Ultimately, these functions should end up in e2ekubectl.

When I wrote the TODO e2ekubectl was a work in progress (it didnt exist really).
I think it would be a good time now to see if these function can be moved there.

Reply:
thanks review.
i will try to move this funcs to e2ekubectl.

Reply:
If you move it directly, it will cause import circular dependency problems.
Keep this TODO temporarily, this migration needs more work。
thanks your comments.
but i think make this funcs private is ok.so I hope to merge this pr.